torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -131,6 +131,62 @@ const std::string shape_compute_functions =
               out.append(li[i])
           return out
 
+        def index_select(self: List[int], dim: int, index: List[int]):
+          dim = maybe_wrap_dim(dim, len(self))
+          numel = multiply_integers(index)
+          assert len(index) <= 1
+          assert dim == 0 or dim < len(self)
+          result_size: List[int] = []
+          for i in range(len(self)):
+            if dim == i:
+              result_size.append(numel)
+            else:
+              result_size.append(self[i])
+          return result_size
+
+        def max_int():
+          return 9223372036854775807
```

Comment:
nit: i guess this is `sys.maxsize`. is there a reason you can't just use that? i can't tell where this python-esque stuff runs...

Reply:
it's not bound yet in torchscript.. i can add a TODO